ureport/settings.py.local

```diff
@@ -0,0 +1,174 @@
+# import our default settings
```

Comment:
Hardly. This is an addition, not a rearrange.

I would argue that this whole settings malarky needs refactored into something more sane.